.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
     - id: black
       language_version: python3
 -   repo: git://github.com/pre-commit/pre-commit-hooks
-    rev: v2.5.0
+    rev: v2.8.2
```

Comment:
Latest is `3.3.0`. Check gen2-dashboards.
On pachamama we have `3.2.0`.

Maybe we are doing something wrong there/here?

Reply:
Check pyproject.toml. There's a ^ just before the pre-commit version, which means that it will only update to versions with Major Release number 2.

On the other repos we use: `*` to denote the version, so this is correct/expected behavior.

To upgrade we'd have to change the version in pyproject.toml

Reply:
I wrote this comment to note that we are applying different strategies in different repos of the same team :D
I'm in favor of unification.